format-as-currency.js

```diff
@@ -74,8 +79,11 @@ angular
 
         if (ngModel.$validators.currency(number)) {
 
-          var formatted = $filter('currency')(number)
-          var specialCharacters = [',', CURRENCY_SYMBOL]
+          var formatted = $filter('currency')(number, suppressCurrencySymbol ? '' : undefined)
+          var specialCharacters = [',']
```

Comment:
i made this guy more generic in https://github.com/bcherny/format-as-currency/commit/118b2f013ef323d7623b583d19cc95f270ff29af